va/va_trace.c

```diff
@@ -772,25 +779,38 @@ void va_TraceInit(VADisplay dpy)
     }
 
     pva_trace->dpy = dpy;
+    pva_trace->ftrace_fd = -1;
 
     pthread_mutex_init(&pva_trace->resource_mutex, NULL);
     pthread_mutex_init(&pva_trace->context_mutex, NULL);
 
+
     if (va_parseConfig("LIBVA_TRACE", &env_value[0]) == 0) {
         pva_trace->fn_log_env = strdup(env_value);
-        trace_ctx->plog_file = start_tracing2log_file(pva_trace);
-        if (trace_ctx->plog_file) {
-            trace_ctx->plog_file_list[0] = trace_ctx->plog_file;
-            va_trace_flag = VA_TRACE_FLAG_LOG;
-
-            va_infoMessage(dpy, "LIBVA_TRACE is on, save log into %s\n",
-                           trace_ctx->plog_file->fn_log);
-        } else
-            va_errorMessage(dpy, "Open file %s failed (%s)\n", env_value, strerror(errno));
+        if (strcmp(env_value, "FTRACE") == 0) {
```

Comment:
reuse LIBVA_TRACE config, set to LIBVA_TRACE = FTRACE to switch into ftrace mode

Reply:
Please update the comments about LIBVA_TRACE in va_trace.c